### PR TITLE
Component/logo

### LINKED ISSUE #13: Logo
Add the Logo component.

Screenshot 2020-12-13 at 22 38 03

View the component on [Figma](https://www.figma.com/file/gGwSe7ilZCUmAIaJyUc4iY/Bristol-Tech-Fair-Design-System?node-id=497%3A3538).

### PR DESCRIPTION
**Pull request by r1903**
Closes #13

This pull request adds the initial version of Logo component. 

Here we can pass the 'width' and 'height' prop to the component to display the different size logo. Also, the logo changes colour on hover.

### CODE REVIEW

File: src/components/Logo/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { LogoWrapper } from './Logo.styles';
+
+const Logo = ({ width, height }) => {
+  return <LogoWrapper width={width} height={height} />;
+};
+
+Logo.propTypes = {
+  width: PropTypes.number,
+  height: PropTypes.number
+};
```
Comment (RobTF9) [inline src/components/Logo/Logo.jsx]: You don't need hegith and width as props, define in styles.

File: src/components/Logo/Logo.styles.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+export const LogoWrapper = styled.div`
+  background-image: url(${LogoRed});
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: ${(props) => (props.width ? `${props.width}rem` : '18.8rem')};
+  height: ${(props) => (props.height ? `${props.height}rem` : '8.2rem')};
+  &:hover {
+    background-image: url(${LogoBlue});
+  }
+`;
```
Comment (RobTF9) [inline src/components/Logo/Logo.styles.js]: Have two options for different colour logs, hover state should be the opposite color for each.

**Review by RobTF9 — CHANGES_REQUESTED** [review]

File: src/components/Logo/Logo.jsx (hunk 1 above)
Comment (r1903) [reply src/components/Logo/Logo.jsx]: I have removed height and width props and hardcoded values in styles.

File: src/components/Logo/Logo.styles.js (hunk 2 above)
Comment (r1903) [reply src/components/Logo/Logo.styles.js]: I have added 'color' prop and on hover it shows the opposite color.

File: src/components/Logo/Logo.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LogoWrapper } from './Logo.styles';
+
+const Logo = ({ color }) => {
```
Comment (RobTF9) [inline src/components/Logo/Logo.jsx]: Add a default value to color:
```
const Logo = ({ color='red' }) => {
...
}
```

**Review by RobTF9 — CHANGES_REQUESTED** [review]
Just add a default value to color, but this is pretty much there.

Great job!

File: src/components/Logo/Logo.jsx (hunk 3 above)
Comment (r1903) [reply src/components/Logo/Logo.jsx]: Thank you for the review. I have done changes.

**Review by RobTF9 — APPROVED** [review]
Looks good, click "Squash and merge" 👍

**Comment by RobTF9** [comment]
Sorry! I accidentally clicked the merge button. Apologies

**Comment by r1903** [comment]
No worries Rob. Thank you very much.

**Pull request merged**